modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -196,13 +198,33 @@ public class UpgradeCatchAllCheck implements ConfigurationBeanDeclaration {
 		KBTemplateModifiedDateComparator.getInstance(false);
 		KBTemplateTitleComparator.getInstance(false);
 		KBTemplateUserNameComparator.getInstance(false);
+		LayoutClassedModelUsageModifiedDateComparator.getInstance(true);
 		LayoutComparator.getInstance(false);
+		LayoutCreateDateComparator.getInstance(true);
+		LayoutModifiedDateComparator.getInstance(false);
+		LayoutPageTemplateEntryCreateDateComparator.getInstance(false);
```

Comment:
@Valmir-unicap The non-parametered constructors of `LayoutPageTemplateEntryCreateDateComparator` and `LayoutPageTemplateEntryModifiedDateComparator` pass `ascending = true` to the other constructor, so these 2 usages should be replaced by `getInstance(true)`.